sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -101,19 +104,24 @@ export interface LogsQueryOptions extends OperationOptions {
     includeQueryStatistics?: boolean;
     includeVisualization?: boolean;
     serverTimeoutInSeconds?: number;
+    throwOnAnyFailure?: boolean;
 }
 
 // @public
 export interface LogsQueryResult {
     error?: ErrorInfo;
+    logsQueryResultStatus?: LogsQueryResultStatus;
```

Comment:
Should this just be `status`? It's already a property of `LogQueryResult`  - the type named `LogQueryResultStatus` is great  IMO, just the property name I am asking about

Reply:
let me check what other languages have called this property name

Reply:
@scottaddie any suggestions? I didn't want to call it "hasError" or "isError" like .NET and Python

Reply:
I'm in agreement with Maor. Let's shorten the property name to `status`.

Reply:
renamed to status and removed the other status which represented HTTP status code